SEED-platform/seed
# Update default meter range selection for BETTER analysis

#### Any background context you want to provide?
The current default behavior of a BETTER analysis is to "Use All Meter Data" and there is no option for cycle select.

#### What's this PR do?
Changes the default behavior to "Select Cycle" where users can choose a cycle to be used as a start and end date range. The current cycle is selected by default.

#### How should this be manually tested?
- Create a new BETTER analysis, the positioning and default behavior should be updated to be on the left and selected.
- Create a new BETTER analysis with a specific cycle. Check analysis results.

#### What are the relevant tickets?
#3776

#### Screenshots (if appropriate)
OLD
![Screenshot 2023-02-02 at 11 34 19 AM](https://user-images.githubusercontent.com/58446472/216419856-7dbbe582-5b6c-457a-9493-7021348f6be2.png)

NEW
![Screenshot 2023-03-31 at 1 48 10 PM](https://user-images.githubusercontent.com/58446472/229215939-21b1e9d7-f9e4-4a04-94e2-e4d37397d857.png)
